frontend/src/components/application/Collection.tsx

```diff
@@ -42,11 +45,38 @@ const ApplicationCollection: FunctionComponent<Props> = ({
     ? applications
     : applications.slice((page - 1) * perPage, page * perPage)
 
+  const refresh = onRefresh ? (
+    <Button onClick={onRefresh} className="float-right">
```

Comment:
I don't think `float-right` is needed, cause you move it with `justify-between` below